### PR TITLE
Fix opensearch release

### PR DESCRIPTION
**Pull request by welpaolo**

### CODE REVIEW

File: .github/workflows/release-tarball.yaml [hunk 1]
```diff
@@ -5,6 +5,7 @@ concurrency:
   cancel-in-progress: true
 
 on:
+  pull_request:
```
Comment (deusebio) [inline .github/workflows/release-tarball.yaml:8]: I thought that we would not release on pull_request, or?

Comment (welpaolo) [reply .github/workflows/release-tarball.yaml]: yes this was just for testing reason

**Review by Mehdi-Bendriss — APPROVED** [review]

File: products.json [hunk 2]
```diff
@@ -18,7 +18,21 @@
...
         "lp-releasing-project": "opensearch-releases",
         "lp-building-repo": "~data-platform/soss/+source/opensearch-build/+git/opensearch-build/",
-        "lp-building-branch-prefix": "lp-2",
+        "lp-building-branch-prefix": "lp-2.8",
+        "lp-consumer-key": "LP_CONSUMER_KEY_SPARK",
+        "lp-access-token": "LP_ACCESS_TOKEN_SPARK",
+        "lp-access-secret": "LP_SECRET_TOKEN_SPARK",
+        "artifactory-url": "https://canonical.jfrog.io/artifactory/dataplatform-opensearch-maven-local/",
+        "artifactory-user": "ARTIFACTORY_USER",
+        "artifactory-token": "ARTIFACTORY_TOKEN",
```
[the remark this answers is not in the document]
Comment (deusebio) [reply products.json:27]: I believe we do. Although we could rename the LP* bits to remove "SPARK". I'll do this straight away... @welpaolo could you update the PR please?

[the remark this answers is not in the document]
Comment (deusebio) [reply products.json:27]: I have left both with and without "_SPARK". Once the PR is merged we can remove the "*_SPARK" secrets

[the remark this answers is not in the document]
Comment (Mehdi-Bendriss) [reply products.json:27]: I think if we move the ownership of all the `build + releases` projects under the `data-platform` team on Launchpad, we no longer need to expose LP related tokens on github, one set should be enough for all projects. And the same should apply with artifactory related credentials  
(we of course lose the flexibility of supporting non data-platform projects)

**Review by deusebio — APPROVED** [review]

**Pull request merged**